nautobot/docs/user-guide/administration/upgrading/from-v1/upgrading-from-nautobot-v1.md

```diff
@@ -36,6 +36,10 @@ All pre-migration checks passed.
 - Nautobot no longer uses or supports the use of `django-mptt`.
 - Nautobot no longer uses or supports the use of `django-rq`.
 
+## App Developer Interface
+
+All imports for app developers should now be sourced from `nautobot.apps.*` or provided models, as this is the supported public interface. This will help ensure proper SemVer is kept in tact and provide a place to add new features without breaking existing integrations.
```

Comment:
```suggestion
All imports for app developers should now be sourced from `nautobot.apps.*` or provided models, as this is the supported public interface. This will help ensure proper SemVer is kept intact and provide a place to add new features without breaking existing integrations.
```